esmvaltool/utils/cmorizers/obs/cmorize_obs_HadCRUT4.ncl

```diff
@@ -47,13 +50,31 @@ begin
   ; Selected variable (standard name)
   VAR = "tas"
```

Comment:
Same here. I think calling a temperature anomaly tas is confusing.

Reply:
In this case it is not the anomaly, but the absolute temperature, because we added the climatology.